docs/stratisd.txt

```diff
@@ -0,0 +1,52 @@
+stratisd(8)
+==========
+
+NAME
+----
+stratisd - Daemon to create and monitor storage pools
+
+SYNOPSIS
+--------
+  stratisd [GLOBAL OPTIONS]
+
+DESCRIPTION
+-----------
+*stratisd* is a daemon that creates, manages, and monitors local storage
```

Comment:
Properly speaking, you should use the indefinite article, "which" instead of "that".